packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -76,11 +76,13 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
     /**
      * Renders the component
      */
-    public render(): React.ReactElement<HTMLInputElement> {
+    public render(): React.ReactElement<HTMLElement> {
         return (
-            <this.tag className={this.generateClassNames()}>
+            <this.tag
+                {...this.unhandledProps()}
```

Comment:
I think this is the right move, but I wonder if there is anything else we need to add as props for the input now that we've moved unhandled props. Thoughts?

Reply:
I'm not sure - I think we're squared away with event listeners because the events will bubble. If there is anything we're missing we can add it fairly easily